### ISSUE TITLE
Upload folder with file not working (first step commit folder created)

### ISSUE THREAD

**Issue by devslimbr** [wangyucode/sftp-upload-action#8]
Hello!
I'm using the sample blank YML.

At first everything works fine.

I created a folder and put a file and committed it. It just created the folder without the file. When I changed the file inside that folder, it was created after the commit

```
on:
  push:
    branches: [ main ]
jobs:
  build:
    runs-on: ubuntu-latest
    defaults:
      run:
        shell: bash
        working-directory: ./
    steps:   

    - name: Checkout
      uses: actions/checkout@v2
      with:
        fetch-depth: 2
    
    - name: restore timestamps
      uses: chetan/git-restore-mtime-action@v1

    - name: SFTP uploader
      uses: wangyucode/sftp-upload-action@v1.4.1
      with:
        host: ${{secrets.HOST}}
        password: ${{secrets.PASS}}
        username: ${{secrets.USER}}
        remoteDir: ${{secrets.DIR}}
```

**Comment by wangyucode** [comment]
I have fixed this issue in the new version `v1.4.5`🚀
please change the action version, and try again.😉
https://github.com/wangyucode/sftp-upload-action/blob/main/.github/workflows/issue-8.yml

Thanks a lot for your reporting.💖

**Comment by devslimbr** [comment]
Thank You!
